docs/tutorials/videos/charts.md

```diff
@@ -0,0 +1,8 @@
+# Taipy Charts to Update Line Types
```

Comment:
I don't like this title, I am trying to understand what it means.

Reply:
The title of the video is "Changing line types using Taipy charts".
That to me is fine (of course I would have dropped the Taipy part).
What about you guys?

Reply:
Well yes, "Changing line types using charts" would work as well, from my point.

Reply:
I prefer this title, yes!